server/db/connection.ts

```diff
@@ -1,31 +1,35 @@
-const sql = require('mssql')
-const colors = require('colors')
+import { ConnectionPool } from 'mssql'
+import { red } from 'colors'
+import config from '../config'
 
 const connectMaster = async () => {
-  const { database } = global.config.db
-  const masterConfig = JSON.parse(JSON.stringify(global.config.db))
-  masterConfig.database = masterConfig.master_database
+  const masterConfig = {
+    ...config.db,
+    database: config.db.master_database,
+  }
+  const { database } = config.db
   try {
-    const pool = await sql.connect(masterConfig)
+    const pool = new ConnectionPool(masterConfig)
+    await pool.connect()
     // prepared statements were not working.
     // also, you set this yourself, so your own fault if you drop all your tables
     await pool
       .request()
       .query(`If(db_id(N'${database}') IS NULL) CREATE DATABASE "${database}"`)
   } catch (err) {
     console.error(
-      'master'.red,
-      'Failed to connect to master database! Check the db.database'.red
+      red('master'),
```

Comment:
Would rather we just removed the colors, and used structured logging, but this is fine for now 🤷‍♀

Reply:
same